enclave/core/sgx/tracee.c

```diff
@@ -9,28 +9,36 @@
 #include "../tracee.h"
 #include "report.h"
 
+static int _is_enclave_debug_allowed = -1;
+
 // Read an enclave's identity attribute to see to if it was signed as an debug
 // enclave
 bool is_enclave_debug_allowed()
 {
-    bool ret = false;
-    oe_sgx_td_t* td = oe_sgx_get_td();
+    if (_is_enclave_debug_allowed != -1)
```

Comment:
Since `is_enclave_debug_allowed` is now performant, the caching for ocall stack stitching can also be removed:
https://github.com/openenclave/openenclave/blob/cd72fd7069488ba6f453c8f5f47bd9fd9a6e6c0d/enclave/core/sgx/calls.c#L634